host/hostthread.h

```diff
@@ -40,15 +40,32 @@ typedef pthread_key_t oe_thread_key;
 typedef INIT_ONCE oe_once_type;
 #define OE_H_ONCE_INITIALIZER INIT_ONCE_STATIC_INIT
 
-typedef DWORD oe_thread;
-
 typedef HANDLE oe_mutex;
 #define OE_H_MUTEX_INITIALIZER INVALID_HANDLE_VALUE
 
 typedef DWORD oe_thread_key;
 
 #endif
 
+/**
+ * Create a platform-specific thread.
+ *
+ * @param func The pointer to the start routine.
```

Comment:
public apis should use [in,out] style doxygen annotation

Reply:
This function is not a public API, like all other functions in the same file.

Reply:
Ah, agreed.   Non-public APIs really should too, but not a blocker.